simplegexf.py

```diff
@@ -312,7 +312,10 @@ def __init__(self, graph, obj, *args, **kwargs):
 
     @property
     def _attvalues(self):
-        return self.obj.data['attvalues']['attvalue']
+        _attvalue = self.obj.data['attvalues']['attvalue']
+        if type(_attvalue).__name__ != 'list':
+            _attvalue = [_attvalue]
+        return _attvalue
```

Comment:
Good catch!

I've discovered that this is a common problem with using `xmltodict`. In some other projects I've implemented a function for dealing with cases such as this:

```python
def ensure_list(var):
    if isinstance(var, list):
        return var
    return [var]
```

However, there is no need to implement a function such as that here unless you can find any other places in the code where this problem exists. Please, feel free to check if you can. ;)

Pro-tips:

- `type(_attvalue).__name__ != 'list'` is an [antipatern](http://docs.quantifiedcode.com/python-anti-patterns/readability/do_not_compare_types_use_isinstance.html)
- Why mutate a variable when you can simply return it?

Reply:
I am a beginner of Python and has learned many Python skills from your code.
I think you are right for simply returning the variable.

Reply:
@mervin0502, Nice to hear. This project was also a small exercise for me in creating an intuitive interface in python. I would further recommend [this PyCon talk by Raymond Hettinger](https://www.youtube.com/watch?v=anrOzOapJ2E)

Would you like to have a try at improving your pull request, or should I do the honours? :)

Reply:
Thank you for your recommendation!

You are better to do that than me.